packages/oruga-next/src/components/input/Input.vue

```diff
@@ -365,6 +365,14 @@ function rightIconClick(event: Event): void {
 const isPasswordVisible = ref(false);
 const inputType = ref(props.type);
 
+const computedInputType = computed(() => {
+    if (props.passwordReveal) {
+        return isPasswordVisible.value ? "input" : "password";
```

Comment:
I think it's "text" instead of "input"

Reply:
hahaha i can't believe i did that! good catch! ironically the default value saved me, it's surprising that worked at all. fixed.